### PR TITLE
[mlagent] Support using of "activated model" in nnstreamer

### PR DESCRIPTION
**Pull request by anyj0527**
- Let the ml_agent related code be built in Tizen
- Let the valid uri scheme be
  `mlagent://model/[name]`  for activated [name] model
  `mlagent://model/[name]/version` for [name] model with given version.
- Let the mock follow naming `model_get` as mlops-agent's interface

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/unittest_mlagent/unittest_mlagent.cc [hunk 1]
```diff
@@ -46,9 +46,40 @@ _init (const std::vector<MockModel> &models = default_models)
...
+constexpr gchar valid_activated_uri_format_literal[] = "mlagent://model/%s";
 constexpr gchar valid_uri_format_literal[] = "mlagent://model/%s/%u";
 constexpr gchar invalid_uri_format_literal[] = "ml-agent://model/%s/%u";
 
+/**
+ * @brief tests of getting activated model path with valid URIs
+ */
+TEST (testMLAgent, GetActivatedModelValidURIs_p)
+{
+  _init ();
```
Comment (niley7464) [inline tests/unittest_mlagent/unittest_mlagent.cc:58]: How about using Test fixture and use `SetUp` function instead of `_init`? (later in another PR)

**Review by niley7464 — APPROVED** [review]
LGTM 👍

**Review by jaeyun-jung — APPROVED** [review]

File: tests/unittest_mlagent/mock_mlagent.h [hunk 2]
```diff
@@ -62,7 +69,19 @@ MockModel *ml_agent_mock_get_model (const gchar *name, const guint version);
...
  */
 gint ml_agent_model_get (
-    const gchar *name, const guint version, gchar **description, GError **err);
+    const gchar *name, const guint version, gchar **description);
+
+/**
+ * @brief Pass the JSON c-string generated by the ML Agent mock class to the caller.
+ * @param[in] name The model's name to retrieve
+ * @param[out] description The JSON c-string, containing the information of the model matching the given name and version
+ * @param[out] err The return location for a recoverable error. This can be NULL.
```
Comment (gichan-jang) [inline tests/unittest_mlagent/mock_mlagent.h:78]: Need to update doxygen tag
```suggestion
 * @param[out] description The JSON c-string, containing the information of the activated model matching the given name.
```

File: tests/unittest_mlagent/mock_mlagent.h [hunk 3]
```diff
@@ -49,7 +49,14 @@ bool ml_agent_mock_add_model (const gchar *name, const gchar *path, const gchar
...
  */
-MockModel *ml_agent_mock_get_model (const gchar *name, const guint version);
+MockModel *ml_agent_mock_model_get (const gchar *name, const guint version);
+
+/**
+ * @brief C-wrapper for the MockModel's method get_model_activated.
+ * @param[in] name The model's name to retrieve
+ * @return A pointer to the model matching the given information. If there is no model possible, NULL is returned.
+ */
+MockModel *ml_agent_mock_model_get_activated (const gchar *name, const guint version);
```
Comment (gichan-jang) [inline tests/unittest_mlagent/mock_mlagent.h:59]:
```suggestion
MockModel *ml_agent_mock_model_get_activated (const gchar *name);
```

File: tests/unittest_mlagent/unittest_mlagent.cc (hunk 1 above)
Comment (anyj0527) [reply tests/unittest_mlagent/unittest_mlagent.cc:58]: Cool. I'll use `SetUp` in later PRs

File: tests/unittest_mlagent/mock_mlagent.h (hunk 2 above)
Comment (anyj0527) [reply tests/unittest_mlagent/mock_mlagent.h]: Thanks! I updated those tags. PTAL

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**